### PR TITLE
Add Primer ViewComponents to home page of tools

### PR DESCRIPTION
**Pull request by srt32**
h/t @ashygee for the new image!

image

### CODE REVIEW

**Comment by yaili** [comment]
@ashygee can you advise? 🙏🏼

**Comment by ashygee** [comment]
Hey @srt32 we don't have an image created for this yet so let me whip something up quickly.

**Review by manuelpuyol — APPROVED** [review]

File: src/pages/index.js [hunk 1]
```diff
@@ -43,6 +44,9 @@ export default function Index() {
             <Promo title="Figma components" image={FigmaImage} href="/design/tools/figma">
               Design layouts, flows, wireframes, and make prototypes with Primer components in Figma.
             </Promo>
+            <Promo title="ViewComponents" image={ViewComponentsImage} href="/view-components">
+              ViewComponents for the Primer Design System.
```
Comment (joelhawksley) [inline src/pages/index.js:48]:
```suggestion
              ViewComponents for use in Rails applications.
```

File: src/pages/index.js [hunk 2]
```diff
@@ -31,12 +31,12 @@ export default function Index() {
             <Promo title="React components" image={ComponentsImage} href="/components">
               Presentational UI components with encapsulated styles and constraint-based theme props.
             </Promo>
+            <Promo title="ViewComponents" image={ViewComponentsImage} href="/view-components">
+              ViewComponents for use in Rails applications.
```
Comment (joelhawksley) [inline src/pages/index.js:35]: I can't make up my mind! How about:

```suggestion
              Build component-driven UI in Rails with ViewComponents.
```

**Pull request merged**